docs/samples/Microsoft.ML.Samples/Dynamic/DataOperations/SaveAndReadFromBinary.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using Microsoft.Data.DataView;
+using Microsoft.ML.SamplesUtils;
+
+namespace Microsoft.ML.Samples.Dynamic.DataOperations
+{
+    public class SaveAndLoadFromBinary
+    {
+        public static void Example()
+        {
+            var mlContext = new MLContext();
+
+            // Get a small dataset as an IEnumerable.
+            IEnumerable<DatasetUtils.SampleTemperatureData> enumerableOfData = DatasetUtils.GetSampleTemperatureData(5);
```

Comment:
>IEnumerable<DatasetUtils.SampleTemperatureData> [](start = 12, length = 47)

`var`?
I'm never sure if we should use `var` or be explicit in samples....

Reply:
I'm a `var` fan, myself, but I'll be more explicit in these examples :)